tests/data/pluto_rotor/vtk.out

```diff
@@ -0,0 +1,31 @@
+0 0.000000e+00 1.000000e-06 0 single_file little rho vx1 vx2 vx3 Bx1 Bx2 Bx3 prs
```

Comment:
Just curious, does the "little" column refer to endianness ? If so, we may have a bug to fix for the case big-endians are used. Idefix _always_ outputs in little endian so it's currently hard-coded in the frontend

Reply:
Yes, it refers to endianness.

Reply:
But I’ve never changed the endianness before, so let me check it tomorrow to figure out how to specify it in PLUTO.

Reply:
Sure, I'll open an issue so we don't loose this thread !